src/storybookCommon/argTypes.ts

```diff
@@ -30,6 +32,16 @@ export const ophanComponentIdArgType = {
     },
 };
 
+export const sectionArgType = {
```

Comment:
To me these section filters feel like they should have their own category (similar to the `core` category), what do you think? I also reckon maybe this `const` should be called something slightly more generic like `pageContextArgTypes` to make it clear where future filtering controls should go.